at_client/test/samples/stream_sender.dart

```diff
@@ -1,22 +1,41 @@
 import 'dart:io';
 
 import 'package:at_client/at_client.dart';
-
+import 'package:at_client/src/stream/at_stream.dart';
+import 'package:at_client/src/stream/at_stream_request.dart';
+import 'package:at_client/src/stream/at_stream_response.dart';
+import 'package:at_utils/at_logger.dart';
 import 'test_util.dart';
 
 void main() async {
+  AtSignLogger.root_level = 'finer';
   try {
     final atsign = '@alice🛠';
     final preference = TestUtil.getAlicePreference();
     var atClientManager = await AtClientManager.getInstance()
         .setCurrentAtSign(atsign, 'wavi', preference);
-    var atClient = atClientManager.atClient;
-    var streamResult =
-        await atClient.stream('@bob🛠', 'cat.jpeg', namespace: 'atmosphere');
-    print(streamResult);
+    var atStreamRequest = AtStreamRequest('@bob🛠', '/home/murali/Pictures/@/cat.jpeg');
+    atStreamRequest.namespace = 'atmosphere';
+    final streamSender =
+        atClientManager.streamService.createStream(StreamType.SEND).sender;
```

Comment:
I was thinking "atClientManager.streamService.createStream(StreamType.SEND)" to return a StreamSender without calling a ".sender". Will discuss in a call.